### DISCUSSION TITLE
the ref in PanelGroup is always null. Not able to call Imperative PanelGroup API because of that.

### DISCUSSION THREAD

**Discussion by ghost** [bvaughn/react-resizable-panels#345] [General]
Hi,

I want to manually resize the panels. I used Imperative Panel API, but not satisfies with that. So I thought to use PanelGroup API. But when pass a ref to it and console log on the same Component, it always show null. the ref is not implemented correctly in PanelGroup.

Kindly check this. Thanks

**Comment by bvaughn** [comment]
> But when pass a ref to it and console log on the same Component, it always show null. the ref is not implemented correctly in PanelGroup.

Sounds like you aren't using the ref correwctly.

> Kindly check this. Thanks

You haven't shared any code or anything that could be checked, so there's no way I can help.
